docs/ruleset_functions/get_SHW_equipment_connected_to_use_type.md

```diff
@@ -0,0 +1,50 @@
+## get_SHW_equipment_connected_to_use_type
+
+Description: This function gets all the SHW equipment connected to a particular SHW use type.  The information is stored in a dictionary where the keys are the ServiceWaterHeatingSpaceOptions2019ASHRAE901 and values are a dictionary giving the ServiceWaterHeatingDistributionSystem, ServiceWaterHeatingEquipment, and Pumps connected to the particular use type  
+
+Inputs:
+- **RMD**
+
+Returns:
+- **shw_and_equip_dict**: A dictionary containing where the keys are the ServiceWaterHeatingSpaceOptions2019ASHRAE901 in the RMD and values are lists of space ids.  Example:  
+{"DORMITORY":{"SHWDistribution":["shwd1","shwd2"], "SHWHeatingEq":["shw_eq1","shw_eq2"], "Pumps":["p1"], "Tanks":["t1"], "Piping":["piping1"], "SolarThermal":[]}}
+
+Function Call:
+
+- get_SHW_types_and_spaces
+- get_obj_by_id
+- get_all_child_SHW_piping_ids
+
+Data Lookup: None
+
+Logic:
+- get the dictionary of SHW types and space ids from the function get_SHW_types_and_spaces: `shw_types_and_uses_dict = shw_and_SHW_use_dict(RMD)`
+- create a blank dictionary: `shw_and_equip_dict = {}`
+- look at space type in shw_types_and_uses_dict: `for shw_use_type in shw_types_and_uses_dict:`
+    - create the blank list for the shw_use_type: `shw_and_equip_dict[shw_use_type] = {"SHWDistribution":[],"SHWHeatingEq":[],"Pumps":[],"Tanks":[], "Piping":[], "SolarThermal":[]}`
+    - look at SHW use: `for use_id in shw_types_and_uses_dict[shw_use_type]:`
+        - get the shw_use using get_obj_by_id: `shw_use = get_obj_by_id(RMD, use_id)`
+        - get the distribution id: `distribution_id = shw_use.served_by_distribution_system`
+        - add the distribution id to the list: `shw_and_equip_dict[shw_use_type]["SHWDistribution"].append(distribution_id)`
+        - get the distribution using get_obj_by_id: `distribution = get_obj_by_id(RMD, distribution_id)`
+        - now we need to get all of the equipment connected to this distribution system.
+        - get the tanks: `tanks = distribution.tanks`
+        - append the tank ids to the dictionary: `for t in tanks:  shw_and_equip_dict[shw_use_type]["Tanks"].append(t.id)`
+        - get the id of the SHW piping: `piping_id = distribution.service_water_piping`
+        - get all the ids of all of the children of the piping: `piping_ids = get_all_child_SHW_piping_ids(RMD, piping_id)`
```

Comment:
Not sure if this accomodates that the distribution system can be connected to multiple ServiceWaterPiping objects.